srv/salt/ceph/apparmor/files/usr.bin.ceph-mgr

```diff
@@ -0,0 +1,41 @@
+#include <tunables/global>
+
+# vim:syntax=apparmor
+# Author: Jan Fajerski <jfajerski@suse.com>
+
+/usr/bin/ceph-mgr {
+  #include <ceph.d/common>
+  #include <abstractions/openssl>
+
+  /usr/bin/ceph-mgr mrix,
+
+  /etc/mime.types r,
+  /bin/bash mr,
+  /bin/bash cx,
+
+  profile /bin/bash {
+    #include <abstractions/base>
+    #include <abstractions/consoles>
+    #include <abstractions/nameservice>
+
+    /bin/bash mr,
+    /usr/bin/uname mrix,
+  }
+
+  /sbin/ldconfig mr,
+  /sbin/ldconfig cx,
+
+  profile /sbin/ldconfig {
+    #include <abstractions/base>
+
+    /sbin/ldconfig mr,
+  }
+
+  /usr/bin/ld.bfd mrix,
+
```

Comment:
Same question for `ldconfig` and `ld`

Reply:
I'm sorry to say I can only shrug and mumble idunno...ceph doesn't work without them.
The profiles were created by filling i the obvious bits, then run a cluster and have apparmor parse its own log file and suggest fixes until the cluster runs. Same for your question above.